templates/client.mustache

```diff
@@ -591,18 +591,6 @@ func addFile(w *multipart.Writer, fieldName, path string) error {
 	return err
 }
 
-// Prevent trying to import "fmt"
-func reportError(format string, a ...interface{}) error {
```

Comment:
I removed `reportError` and `newStrictDecoder` as they exist in `utils.mustache`.
I pulled the templates out of generator using `openapi-generator author template -g go -o out` and saw these functions in utils & client templates